### PR TITLE
[Merged by Bors] - chore: fix some to_additive technical debt

### PR DESCRIPTION
**Pull request by fpvandoorn**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Filter/Pointwise.lean [hunk 1]
```diff
@@ -598,9 +598,9 @@ protected theorem mul_eq_one_iff : f * g = 1 ↔ ∃ a b, f = pure a ∧ g = pur
     rw [pure_mul_pure, h, pure_one]
 
 /-- `Filter α` is a division monoid under pointwise operations if `α` is. -/
-@[to_additive subtractionMonoid "`Filter α` is a subtraction monoid under pointwise operations if
+@[to_additive? subtractionMonoid "`Filter α` is a subtraction monoid under pointwise operations if
  `α` is."]
--- Porting note: `to_additive` guessed `divisionAddMonoid`
+-- check if fixed
```
Comment (grunweg) [inline Mathlib/Order/Filter/Pointwise.lean:603]: Can you check this?

**Review by grunweg — APPROVED** [review]
Looks very plausible. Thank you for doing this!

**Comment by grunweg** [comment]
Looks reasonable to me!
maintainer merge

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**